dp/generate_climos.py

```diff
@@ -39,6 +39,86 @@
 cdo = Cdo()
 
 
+def has_climo_periods(input_file, climo):
```

Comment:
This is a bit of a nit-pick but I think the name of this method suggests the output will be a `boolean`.  Something like `get_climo_periods` may be more appropriate.